cql3/statements/modification_statement.cc

```diff
@@ -439,13 +441,30 @@ modification_statement::process_where_clause(data_dictionary::database db, expr:
     }
 }
 
+::shared_ptr<strongly_consistent_modification_statement>
+modification_statement::prepare_for_raft() const {
```

Comment:
`prepare_for_raft` should eventually become a pure virtual function in `modification_statement` and it should be implemented in every type of `modification_statement`.

Please live a FIXME which tells us that, so the reader of the code knows this state is temporary (there is no inherent reason that we throw unsupported here other than it's WIP).

Reply:
Done